packages/visual-editor/src/components/Dropzone/EditorBlock.tsx

```diff
@@ -87,6 +95,36 @@ export const EditorBlock: React.FC<EditorBlockProps> = ({
     });
   };
 
+  if (isComponentMissing || !definition || !elementToRender) {
+    const renderMissingComponentPlacehoder = () => (
+      <div
+        style={{
+          border: '1px solid red',
+          width: '100%',
+          height: '100%',
+        }}>
+        Missing component &lt;{node.data.blockId}&gt;
```

Comment:
Putting the angle brackets around the name of the component might make the devs think this is how it is referenced in React, which would be incorrect. I wonder if we should just show the name here minus the brackets

Reply:
✅ FIXED (in new PR)